transports/quic/Cargo.toml

```diff
@@ -15,7 +15,7 @@ futures = { workspace = true }
 futures-timer = "3.0.3"
 if-watch = "3.2.0"
 libp2p-core = { workspace = true }
-libp2p-tls = { workspace = true }
+libp2p-tls = "0.3.0"
```

Comment:
temporarily downgrade `quic`'s `libp2p-tls` dependency to isolate this PR, upgrade back on https://github.com/libp2p/rust-libp2p/pull/5316